Combobox/themes/bootstrap/Combobox.less

```diff
@@ -13,11 +13,12 @@
 }
 
 .d-combobox-arrow-styles() {
-	font-size: .7em;
-	right: 1em;
-	top: 50%;
-	height: 1em;
-	margin-top: -0.6em;
+	top: 0;
+	right: 0;
+	width: 28px;
+	text-align: center;
+	height: 28px;
```

Comment:
Using 1em gives me around 10px because of the font-size .7em to fix the arrow size. I can calculate how much ems to use the same measurement under .7em font-size.